riotfile.py

```diff
@@ -1874,43 +1874,28 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 "aiopg": "~=0.16.0",
                 "pytest-randomly": latest,
             },
-            # venvs=[
-            # FIXME: tests fail on aiopg 1.x
-            # Venv(
-            #     # aiopg dropped support for Python 3.5 in 1.1
-            #     pys="3.5",
-            #     pkgs={
-            #         "aiopg": ["~=0.16.0", "~=1.0"],
```

Comment:
How far back do we want to support aiopg? It seems like this PR makes things work with recent versions but I think the changes will likely break support for older releases given that we've changed the interface we're patching.

To give some perspective:

* 0.16.0: Jan 15 2019
* 1.2.0: Mar 23 2021
* 1.3.0: June 30 2021
* 1.4.0: Oct 26 2022

https://pypi.org/project/aiopg/#history

Reply:
Here are our goals for version support: https://github.com/DataDog/dd-trace-py/blob/b2a4fb1b8686237445151fa1f571fbb45c4e594b/docs/contributing-integrations.rst#what-versions-of-a-given-library-should-an-integration-support

In the case of aiopg, looks like this means we want to test 1.0 and 1.4.